partner/app/tests/test_app.py

```diff
@@ -24,7 +24,7 @@ def test_read_tilesp(client):
             "sub1": "123456789",
             "sub2": "placement1",
             "country-code": "US",
-            "region-code": "NY",
+            "region-code": "US-NY",
```

Comment:
Please keep the example values from the API specification in this happy-path test.

Reply:
Do you mean that I should use the exact example values from the`main.py` query param here? Like `US-NY` at both the places?

Reply:
No. The `test_read_tilesp` test makes sure that the mock partner API works as designed and handles the example values from the API specification PDF for each of the query parameters of the `/tilesp` endpoint correctly. So we need to keep this as `"NY"`.